README.md

```diff
@@ -7,12 +7,57 @@
 
 A Terraform Module to integrate Amazon Elastic Kubernetes Service (EKS) with Lacework.
 
+**Currently, this module is limited to integrating clusters on a per-region basis, due to a limitation of the 
+`aws_cloudwatch_log_subscription_filter` Terraform resource.**
+
+### Pre-requisite
+Audit logging must be enabled on the cluster(s) which you wish to integrate. This can be done via the AWS CLI using the following command:
+```bash
+aws eks --region <region-code> update-cluster-config --name <prod> \
+--logging '{"clusterLogging":[{"types":["audit"],"enabled":true}]}'
+```
+
+## Resources created
+- SNS topic
+- Topic policy
+- S3 bucket
+- S3 bucket notification
+- Firehose
+- Firehose IAM role & policy
+- Cross account IAM role & policy
+- Cloudwatch IAM role & policy
+- Cloudwatch subscription filter
+
 ## Inputs
 
-| Name | Description | Type | Default | Required |
-|------|-------------|------|---------|----------|
+| Name                             | Description                                                                                                          | Type           | Default                     | Required |
+| -------------------------------- | -------------------------------------------------------------------------------------------------------------------- | -------------  | --------------------------- | :------: |
+| bucket_enable_encryption         | Set this to `true` to enable encryption on a created S3 bucket                                                       | `bool`         | `false`                     |    no    |
+| bucket_enable_mfa_delete         | Set this to `true` to require MFA for object deletion (Requires versioning)                                          | `bool`         | `false`                     |    no    |
+| bucket_enable_versioning         | Set this to `true` to enable access versioning on a created S3 bucket                                                | `bool`         | `false`                     |    no    |
```

Comment:
Use `bucket_versioning_enabled` instead since we use it already in our CloudTrail module